tracer/qemu_runner.py

```diff
@@ -0,0 +1,400 @@
+import os
+import angr
+import time
+import shutil
+import signal
+import socket
+import logging
+import resource
+import tempfile
+import subprocess
+import contextlib
+
+from elftools.elf.enums import ENUM_EI_OSABI
+from .tracerpov import TracerPoV
+from .tracer import TracerEnvironmentError
+from .tinycore import TinyCore
+from .runner import Runner
+
+l = logging.getLogger("tracer.QEMURunner")
+
+try:
+    import shellphish_qemu
+except ImportError:
+    l.error("Unable to import shellphish_qemu, which is required by QEMURunner. Please install it before proceeding.")
+    assert False
```

Comment:
Maybe replace with `raise ImportError("Unable to import shellphish_qemu, which is required...")`?